src/status_im/contexts/wallet/common/wizard/utils.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.contexts.wallet.common.wizard.utils
+  (:require [status-im.contexts.wallet.bridge.flow-config :as wallet-bridge-flow]
+            [status-im.contexts.wallet.send.flow-config :as wallet-send-flow]))
+
+(defn wizard-find-next-screen
```

Comment:
Now when we don't use the `wizard-find-next-screen` this change can be reverted and become the private function again